docs/zh/latest/control-api.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Control API
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+control API 可以被用来:
+
+* 暴露APISIX内部状态信息
+* 控制单个APISIX的数据平面的行为
+
+默认情况下,control API是启用的,监听`127.0.0.1:9090`.你可以通过修改`apisix/conf/config.yaml`中的control部分来更改设置,如下:
```

Comment:
```suggestion
默认情况下，control API 是启用的，监听 `127.0.0.1:9090`。你可以通过修改 `apisix/conf/config.yaml` 中的control 部分来更改设置，如下:
```

And please fix other similar errors.

- Chinese should be preceded by spaces between English, numbers and other special symbols.
- Use Chinese punctuation.